prime/src/components/borrow/actions/AloeRepayActionCard.tsx

```diff
@@ -55,7 +56,7 @@ export function AloeRepayActionCard(prop: ActionCardProps) {
         actionId: ActionID.REPAY,
         actionArgs: value === '' ? undefined : getRepayActionArgs(token0, amount0, token1, amount1),
         operator(operand) {
-          return repayOperator(operand, selectedToken, GN.max(amount0, amount1));
+          return repayOperator(operand, token, parsedValue);
```

Comment:
How long was this `selectedToken` instead of `token`? Have we had this bug forever?